File: fc-chat/demo/part3/ch3/05-1-message-system-add-channel/src/main/java/net/prostars/messagesystem/dto/websocket/inbound/FetchChannelsRequest.java
package net.prostars.messagesystem.dto.websocket.inbound;

import com.fasterxml.jackson.annotation.JsonCreator;
import net.prostars.messagesystem.constant.MessageType;

public class FetchChannelsRequest extends BaseRequest {

  @JsonCreator
  public FetchChannelsRequest() {
    super(MessageType.FETCH_CHANNELS_REQUEST);
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/inbound/FetchChannelsRequest.kt
package com.yscorp.withpush.messagesystem.dto.websocket.inbound

import com.yscorp.withpush.messagesystem.constant.MessageType

class FetchChannelsRequest @com.fasterxml.jackson.annotation.JsonCreator constructor() :
    BaseRequest(MessageType.FETCH_CHANNELS_REQUEST)
